aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -61,6 +63,7 @@ module aptos_framework::genesis {
 
         // Give the decentralized on-chain governance control over the core framework account.
         aptos_governance::store_signer_cap(&aptos_framework_account, @aptos_framework, framework_signer_cap);
+        aptos_governance::store_signer_cap(&aptos_token_account, @0x3, aptos_token_signer_cap);
```

Comment:
same

Reply:
it is more clear to store signer_cap per account. If I have to squash everything in one function call, each parameter is a vector and need to align the accounts in different vector parameters.